casdk-docs/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "casdk-docs",
+  "version": "0.0.0",
+  "private": true,
+  "scripts": {
+    "docusaurus": "docusaurus",
+    "start": "docusaurus start",
```

Comment:
Possibly can edit docusaurs scripts to add the required copy command for copying samples docs: `cp -r ./samples/ ./casdk-docs/docs/ && <OTHER_COMMANDS>`

Reply:
Cool will give it a go soon. Thanks